Assets/VRM/UniVRM/Scripts/Format/VRMExporter.cs

```diff
@@ -197,13 +196,6 @@ public static void _Export(glTF gltf, VRMExporter exporter, GameObject go)
                             gltf.extensions.VRM.firstPerson.lookAtVerticalUp.Apply(blendShapeApplyer.VerticalUp);
                         }
                     }
-                    else if (lookAt != null)
-                    {
-                        gltf.extensions.VRM.firstPerson.lookAtHorizontalInner.Apply(lookAt.HorizontalInner);
```

Comment:
I think that throw an exception which tells that ”this feature is not supported” is better than removing code with no notifications for users.What do you think about this?

Reply:
VRMLookAt is a very early version of VRMLookAtHead and VRMBoneApplyer. It is older than first release 0.29. 
I forgot to remove it.

Reply:
Thanks, I see! If no one uses this component currently, I think it's ok to remove code without attention.